micro-benchmarks/nccl-tests/slurm/topology-aware-nccl-tests/submit_nccl_test_container.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+# Script to submit comprehensive NCCL tests with 2 and 4 nodes
+# Tests all collective operations with different split masks
+
+set -e
+
+# Configuration
+NODE_COUNTS=(2 4 8 16)
+APPS_PATH="/fsxl"
```

Comment:
```suggestion
APPS_PATH="${APPS_PATH:-/fsxl}"
```
as we want to respect already-exported-environment-variable (if we have).